src/prog.js

```diff
@@ -33,6 +33,11 @@ export default handler => {
 		.option('--name', 'Specify name exposed in UMD builds')
 		.option('--cwd', 'Use an alternative working directory', '.')
 		.option('--sourcemap', 'Generate source map', true)
+		.option(
+			'--css-modules',
+			'Turns on css-modules for all .css imports. Passing a string will override the generatedScopeName. eg --css-modules="_[hash]"',
+			null,
```

Comment:
setting the default to true would make the conditional above more readable 🙃

Reply:
@ForsakenHarmony it's a tri-state though. `--css-modules true` turns on CSS modules for all `.css` files, whereas the default `null` value turns on CSS modules only for `.module.css` files.

I wonder though if you're pointing out that there might not be much reason to offer `--css-modules false`? (which disables css modules for `.module.css` files)